packages/expo-camera/src/Camera.types.ts

```diff
@@ -94,6 +94,29 @@ export enum ImageType {
   jpg = 'jpg',
 }
 
+export enum VideoCodec {
```

Comment:
let's add a docblock here about video codec to save @Simek the hassle of copying over from docs when he gets to this module for auto gen docs